src/marqo/tensor_search/tensor_search.py

```diff
@@ -856,16 +860,21 @@ def _add_documents_structured(config: Config, add_docs_params: AddDocsParams, ma
                                 chunk_id = f"{chunk_time}"
                                 chunks.append(chunk_id)
 
-                                vector = s2_inference.vectorise(
-                                    model_name=marqo_index.model.name,
-                                    content=[media_chunk['tensor']],  # Wrap in list as vectorise expects an iterable
-                                    model_properties=marqo_index.model.get_properties(),
-                                    device=add_docs_params.device,
-                                    normalize_embeddings=marqo_index.normalize_embeddings,
-                                    infer=True,
-                                    model_auth=add_docs_params.model_auth,
-                                    modality=Modality.VIDEO if marqo_field.type == FieldType.VideoPointer else Modality.AUDIO
-                                )
+                                start_time = timer()
+                                with RequestMetricsStore.for_request().time(f"add_documents.create_vectors"):
```

Comment:
Should we wrap this outside the for loop to capture the time for each video/audio field instead of each chunk?

Reply:
What's the behavior in 2.11? That could be too verbose?

Reply:
in 2.11, we don't have video or audio fields. For image or text fields, we record this for each individual field.
For multi-modal combo fields, we record this metric for each type of modality under `create_vectors` not `add_documents.create_vectors`. So it's quite inconsistent.